### DISCUSSION TITLE
The code commit widget isn’t working. Always says “sign in”. Tapping it brings me to app where I’m already signed in.

### DISCUSSION THREAD

**Discussion by gonzaloandresoto** [community/community#72628] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.136.0 (113105320)
Device: iPhone 13 Pro Max (iOS 17.0.3)
</details>

**Comment by Iron-Ham** [comment]
Hi @gonzaloandresoto! Thank you for providing the report. I'll make an internal issue to track it. 

Thanks,
Hesham

**Comment by demirdovenikbal** [comment]
yeah i was having the same issue and deleted, signed out and downloaded the app back and it worked for me.
